lib/pbench/test/unit/server/test_sync.py

```diff
@@ -0,0 +1,90 @@
+from pbench.server.database.models.datasets import Dataset, Metadata
+from pbench.server.sync import Operation, Sync
+
+
+class TestSync:
+    def test_construct(self, make_logger):
+        sync = Sync(make_logger, "test")
+        assert sync.logger is make_logger
+        assert sync.component == "test"
```

Comment:
My concern about testing constructors is that they typically aren't testing the object's interface (in proper OO, there are supposed to be getters and setters for attributes in order to encapsulate things properly), and it's easy for the tests to get stale (because adding attributes doesn't prompt one to update the test).  But, I guess it doesn't hurt...much.  🥴